--- a/src/main/java/app/attestation/server/AttestationServer.java
+++ b/src/main/java/app/attestation/server/AttestationServer.java
@@ -202,7 +202,7 @@ public class AttestationServer {
         new Thread(new AlertDispatcher()).start();
         new Thread(new Maintenance()).start();
 
-        final HttpServer server = HttpServer.create(new InetSocketAddress("localhost", 8080), 0);
+        final HttpServer server = HttpServer.create(new InetSocketAddress("0.0.0.0", 8080), 0);
         server.createContext("/submit", new SubmitHandler());
         server.createContext("/create_account", new CreateAccountHandler());
         server.createContext("/change_password", new ChangePasswordHandler());
