--- a/src/main/java/app/attestation/server/AttestationProtocol.java
+++ b/src/main/java/app/attestation/server/AttestationProtocol.java
@@ -154,9 +154,9 @@ class AttestationProtocol {
     private static final String ATTESTATION_APP_PACKAGE_NAME = "app.attestation.auditor";
     private static final int ATTESTATION_APP_MINIMUM_VERSION = 1;
     private static final String ATTESTATION_APP_SIGNATURE_DIGEST_DEBUG =
-            "17727D8B61D55A864936B1A7B4A2554A15151F32EBCF44CDAA6E6C3258231890";
+            "F2B34C92225311DCD01D9FEBCE9851EFB97416346518C54CB6BC8D7020781514";
     private static final String ATTESTATION_APP_SIGNATURE_DIGEST_RELEASE =
-            "990E04F0864B19F14F84E0E432F7A393F297AB105A22C1E1B10B442A4A62C42C";
+            "F2B34C92225311DCD01D9FEBCE9851EFB97416346518C54CB6BC8D7020781514";
     private static final int OS_VERSION_MINIMUM = 80000;
     private static final int OS_PATCH_LEVEL_MINIMUM = 201801;
     private static final int VENDOR_PATCH_LEVEL_MINIMUM = 201809;
